### ISSUE TITLE
issue in launching the controller node

### ISSUE THREAD

**Issue by Tejas-Baid** [deepboliya/pendulum-problems-ros2#2]
after creating the controller node i realized that i need to add it to the launcher file as well.but after doing so i was unable to launch the file as the launcher was somehow not able to locate it.
These are my additions in the launch file:
dynamics = Node(
        package = 'single_inverted',
        executable = 'dynamics_sim'
    )
    
    control = Node(
        package = 'single_inverted',
        executable = 'controller'
    )

    return LaunchDescription([
        visualizer,
        dynamics,
        control
    ])

These are my changes in setup.py
 entry_points={
        'console_scripts': [
            'dynamics_sim = single_inverted.dynamics_sim:main',
            'controller = single_inverted.controller:main'
        ],
this is the error i am facing:
[INFO] [launch]: Default logging verbosity is set to INFO
Task exception was never retrieved
future: <Task finished name='Task-2' coro=<LaunchService._process_one_event() done, defined at /opt/ros/foxy/lib/python3.8/site-packages/launch/launch_service.py:226> exception=SubstitutionFailure("executable 'controller' not found on the libexec directory '/home/tejas/Desktop/PendulumAss/assWS/install/single_inverted/lib/single_inverted' ")>

**Comment by deepboliya** [comment]
Are you able to run your node separately after launching the original launch file?

**Comment by Tejas-Baid** [comment]
yes, if i launch the launch file without 'control' node in  launchDescription , I am able to launch it . On the side i can manually run the controller node and the expected output on the simulation and the terminal is also shown

**Comment by AyuwanC** [comment]
Did you do colcon build of the whole workspace again?
